src/transformers/utils/quantization_config.py

```diff
@@ -215,11 +209,8 @@ def __init__(
         self,
         nbits: int = 4,
         group_size: int = 64,
-        quant_zero: bool = True,
-        quant_scale: bool = False,
-        offload_meta: bool = False,
         view_as_float: bool = False,
-        axis: int = 0,
```

Comment:
Breaking change here. Is this the recommended value now ?

Reply:
Yes, I have been planning to do it for a long time, because only `axis=1` supports the fast backends like `torchao` and `bitblas`, so the new default value is `axis=1` instead of `axis=0`

Reply:
Well, I will leave it to you but I think it will be great to let the user know that axis have been set to 1 since it supports fast backends. How I would do is to set axis = None and in the post_init, if axis is None, we set it to 1 + add a `logger.info`

Reply:
`axis` is required and it should be either 0 or 1 actually. It's just that before it was 0 and now it's 1.
What should the `logger.info` say?

Reply:
I think the diff doesn't show exactly what's going on: https://github.com/huggingface/transformers/blob/ff982b3b2c8e9f4580ad0a32b90c40b3dcae5985/src/transformers/utils/quantization_config.py#L186-L217

Reply:
see new review